Sources/SQLite/SQLite.swift

```diff
@@ -31,20 +31,21 @@ public class SQLite {
     */
     public init(path: String) throws {
         let options = SQLITE_OPEN_CREATE | SQLITE_OPEN_READWRITE | SQLITE_OPEN_FULLMUTEX
-        if sqlite3_open_v2(path, &database, options, nil) != SQLITE_OK {
-            throw SQLiteError.connection(database?.errorMessage ?? "")
-        }
+
+        let status = sqlite3_open_v2(path, &database, options, nil)
+
+        StatusError.check(with: status, msg: database?.errorMessage ?? "")
```

Comment:
should this not be `try`? It won't compile for me without it...

Reply:
That is right.

Reply:
How did pass CI?

Reply:
(PS don't mean to sound accusatory! Typos are easy but we should just make sure it shouldn't happen again!)

Reply:
That is unknown. In addition, it compiled to me before pushing. Strange. Check #25 with the fix.

Reply:
I agree.

Reply:
My money's on the good old friend that is derived data! 😁

Reply:
Thanks for the feedback!